DEVELOPMENT.md

```diff
@@ -1,6 +1,6 @@
 ## Development Guidelines
 
-This document intends to establish guidelines which build a transparent, open mechanism for deciding how to evolve the OpenAPI Specification. The Open API Technical Contributor Board will initially follow these processes when merging changes from external contributors or from the TCB itself. This guideline document will be adjusted as practicality dictates.
+This document establishes guidelines which build a transparent, open mechanism for deciding how to evolve the OpenAPI Specification. The OpenAPI Technical Developer Community (TDC) will initially follow these processes when merging changes. This guideline document will be adjusted as practicality dictates.
```

Comment:
Use `[Technical Developer Community](https://www.openapis.org/participate/how-to-contribute/governance#TDC) (TDC)` to render as [Technical Developer Community](https://www.openapis.org/participate/how-to-contribute/governance#TDC) (TDC)